### PR TITLE
Navigation Bar created

### PR DESCRIPTION
**Pull request by rutu14**

### CODE REVIEW

File: css/nav.css [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    padding: 0 0.8rem 0 2rem;
+    box-shadow: 0 2px 5px 0 rgb(0 0 0 / 5%), 0 2px 10px 0 rgb(0 0 0 / 5%); 
+}
+
+.header-logo{
+    display: flex;
+    align-items: center;
+}
+
+.brand-name{
```
Comment (itsmebipul00) [inline css/nav.css:15]: you could have defined all your font-families in the root tag with a variable name

Comment (itsmebipul00) [reply css/nav.css:15]: and use them again and again wherever u want

**Review by itsmebipul00 — APPROVED** [review]

**Review by sudipkundu999 — APPROVED** [review]
Looks good to me.
You may use px for less than 1rem.

File: css/nav.css [hunk 2]
```diff
@@ -0,0 +1,99 @@
+.header {
+    height: 6.4rem;
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+    padding: 0 0.8rem 0 2rem;
+    box-shadow: 0 2px 5px 0 rgb(0 0 0 / 5%), 0 2px 10px 0 rgb(0 0 0 / 5%);
```
Comment (zoef-yt) [inline css/nav.css]: Use var for colours.

**Review by zoef-yt — CHANGES_REQUESTED** [review]

File: css/nav.css (hunk 1 above)
Comment (rutu14) [reply css/nav.css:15]: Sure.

File: css/nav.css (hunk 2 above)
Comment (rutu14) [reply css/nav.css]: Okay

**Review by zoef-yt — APPROVED** [review]
Good to merge.

**Pull request merged**